pom.xml

```diff
@@ -490,6 +491,11 @@
                 <artifactId>auto-value-javabean</artifactId>
                 <version>${auto-value-javabean.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.posthog.java</groupId>
+                <artifactId>posthog</artifactId>
```

Comment:
The library looks a bit funky. Using `System.out.println` and `e.printStackTrace()` for logging, for example. :grimacing: 

https://github.com/PostHog/posthog-java/blob/e230c6417088d26a03c00905d3c301b341eff624/posthog/src/main/java/com/posthog/java/HttpSender.java#L98

I wonder if it's worth including or if we should rather implement our own small client based on our OkHttp client. Not sure how much work the client saves us. I don't think we need the client's queueing mechanism.

Reply:
I see no way to specify a proxy for the client, so that is already a good reason to roll our own, unfortunately. My only concern are future changes that we cannot mitigate by just bumping the client, but maybe at that point the client is  in a better shape, so we can switch.

Reply:
Thanks!

Their HTTP API should be rather stable. If that's not the case, it would be pretty painful for everyone using the product. :smile: